lib/faker/datetime.ex

```diff
@@ -0,0 +1,45 @@
+if Version.match?(System.version(), ">= 1.3.0") do
+  defmodule Faker.DateTime do
+    @microseconds_per_day 86400000000
+
+    @doc """
+    Returns a random date in the past up to N days, today not included
+    """
+    @spec backward(integer) :: DateTime.t
+    def backward(days) do
+      forward(-days)
+    end
+
+    @doc """
+    Returns a random date in the future up to N days, today not included
+    """
+    @spec forward(integer) :: DateTime.t
+    def forward(days) do
+      unix_from = DateTime.utc_now() |> DateTime.to_unix(:microseconds)
+
+      unix_between(unix_from, days)
+    end
+
+    @doc """
+    Returns a random date between two dates, start date and end date not
+    included
+    """
+    @spec between(DateTime.t, DateTime.t) :: DateTime.t
+    def between(from, to) do
+      unix_from = DateTime.to_unix(from, :microseconds)
+      unix_to = DateTime.to_unix(to, :microseconds)
+      days = round(Float.floor((unix_to - unix_from) / @microseconds_per_day))
+
+      unix_between(unix_from, days)
```

Comment:
While the function implies it returns a random DateTime object between X and Y, the returned time of the datetime seems to be tied to the from-datetime since you add a random number of days to the `from`. Is that intentional? Why not add a random number of microseconds to the timestamp between 0 and `unix_to - unix_from`?

This has the added bonus you can remove `@microseconds_per_day`, since many days do not have exactly `86400000000` microseconds in them.